test/hoya/hoya_coupled.cpp

```diff
@@ -58,6 +58,6 @@ int main() {
     std::shared_ptr<cadmium::dynamic::modeling::coupled<TIME>> t = std::make_shared<grid_coupled<TIME, sir, mc>>(test);
```

Comment:
The file is named `hoya_coupled.cpp`. You should rename it to match the CO2 model